Lib/Filter/ScssPHP.php

```diff
@@ -30,11 +30,16 @@ public function input($filename, $input) {
 		if (substr($filename, strlen($this->_settings['ext']) * -1) !== $this->_settings['ext']) {
 			return $input;
 		}
-		App::import('Vendor', 'scssc', array('file' => $this->_settings['path']));
-		if (!class_exists('scssc')) {
-			throw new Exception(sprintf('Cannot not load filter class "%s".', 'scssc'));
+		require($this->_settings['path']);
+		if (class_exists('scssc')) {
+			// in order to keep compatibility with older ScssPHP versions
+			$sc = new scssc();
+			$sc->addImportPath(dirname($filename));
+			return $sc->compile($input);
```

Comment:
These 2 lines are the same in both implementations. The conditional part seems to be how the scssc class is constructed.